nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/commands/IndexingCommand.java

```diff
@@ -344,4 +340,8 @@ public long getOrder() {
     public void setOrder(long order) {
         this.order = order;
     }
+
+    public enum Type {
+        INSERT, UPDATE, UPDATE_SECURITY, DELETE, UPDATE_DIRECT_CHILDREN,
+    }
```

Comment:
I find it strange to put the enum at the end of the class, it was all right where it was.

Reply:
yes I did a rearrange code by mistake, if I am not mistaken the Nuxeo code style put enums after methods weird